### DISCUSSION TITLE
Toggle the theme based on system UI semi-automatically. lol

### DISCUSSION THREAD

**Discussion by MJLiiii** [NvChad/NvChad#3022] [Ideas]
![image](https://github.com/user-attachments/assets/ef66552f-1cd1-41fd-b4d2-c5e92ef6025e)
I dont' find better way to switch theme with UI automatically, so this is a good way. Maybe it will set "onedark" when save the chadrc.lua file.
If anyone have beeter way, pla tell me. lol

**Comment by siduck** [comment]
@Snuff-red  that is the way actually. Read :h nvui.base46.api for more recipes
